docs/tutorials/join-dal-baker/run-dal-node.md

```diff
@@ -15,21 +15,7 @@ octez-dal-node run >> "$HOME/octez-dal-node.log" 2>&1
 
 This, too, may take some time to launch the first time because it needs to generate a new identity file, this time for the DAL network.
 
-The DAL node connects to the DAL network but it is not yet receiving shards.
+The DAL node connects to the DAL network but it is not yet receiving data.
 
-DAL nodes share shards and information about them over a peer-to-peer pub/sub network built on the Gossipsub protocol.
-As layer 1 assigns shards to the bakers, the Gossipsub network manages topics that DAL nodes can subscribe to.
-For example, if a user submits data to slot 1, the network has a list of topics: a topic to identify the slot 1 shards assigned to baker A, a topic to identify the slot 1 shards assigned to baker B, and so on for all the bakers that are assigned shards from slot 1.
-
-Then, the baker daemon automatically asks the DAL node to subscribe to the topic that provides the shards that it is assigned to.
-You can see the topics that the DAL node is subscribed to by running this RPC call:
-
-```bash
-curl http://localhost:10732/p2p/gossipsub/topics
-```
-
-If you are using the `tezos/tezos` Docker image, you can install the `curl` program by running the command `sudo apk add curl`.
-
-In the results, each topic contains a shard and the address of the baker that is assigned to it.
-Now you can run a baker to verify these shards.
-Continue to [Step 5: Run an Octez baking daemon on Weeklynet](./run-baker).
+Now that you have a DAL node running, you can start a baking daemon that uses that DAL node.
```

Comment:
Maybe we should had a comment to check the DAL node is bootstrapped (should be quick) when there is a log about L1 node's heads.

By the way, something that could be useful too is to say that the DAL node waits for an L1 node block to be finalized, so there is a delay of 2 blocks.

Reply:
We could also add a check here to the `/p2p/gossipsub/connections` RPC to check your node is connected to the bootstrap node. Since we do not give any profile yet, it may not be connected to any topics of interest at the moment, so the `topics` RPC could return an empty array.

Reply:
Added info on how to check that the DAL node is bootstrapped in b88dcb7b3f633a72cd4ec56eec9349edf0ca6844.

Reply:
Added:

> The DAL node waits for blocks to be finalized, so this log lags 2 blocks behind the layer 1 node's log.

Reply:
> We could also add a check here to the `/p2p/gossipsub/connections` RPC to check your node is connected to the bootstrap node. Since we do not give any profile yet, it may not be connected to any topics of interest at the moment, so the `topics` RPC could return an empty array.

I find it the other way around: I get data for `curl http://localhost:10732/p2p/gossipsub/topics` but an empty array for `curl http://localhost:10732/p2p/gossipsub/connections`.

Reply:
Getting data for `/connections` and `/topics` now.